crates/arroyo-api/migrations/V6__add_cluster_table.sql

```diff
@@ -1,3 +1,4 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
```

Comment:
This approach won't work unfortunately; migrations are hashed and that's used to determine which ones have been applied. So if we modify an existing migration that will break it for everyone who's already run them.

I think that instead we could run this on the database before invoking the migrations in main:

https://github.com/ArroyoSystems/arroyo/blob/29e96b03e6f612feecfafc917ef40cc5aa282bdb/crates/arroyo/src/main.rs#L375

Reply:
yeah, I have fix it now.